hexrd/core/fitting/calibration/instrument.py

```diff
@@ -4,18 +4,8 @@
 import lmfit
 import numpy as np
 
-from .lmfit_param_handling import (
-    add_engineering_constraints,
-    create_instr_params,
-    DEFAULT_EULER_CONVENTION,
-    update_instrument_from_params,
-    validate_params_list,
-)
-from .relative_constraints import (
-    create_relative_constraints,
-    RelativeConstraints,
-    RelativeConstraintsType,
-)
+from .lmfit_param_handling import add_engineering_constraints, create_instr_params, DEFAULT_EULER_CONVENTION, update_instrument_from_params, validate_params_list
+from hexrd.core.fitting.calibration.relative_constraints import create_relative_constraints, RelativeConstraints, RelativeConstraintsType
```

Comment:
You might want to run `black` on the files before we merge this one.

Reply:
Ran black on everything 👍